packages/template-typescript-minimal/jest.config.js

```diff
@@ -1,11 +1,22 @@
 /*
- * Copyright (c) 2021, salesforce.com, inc.
+ * Copyright (c) 2022, Salesforce, Inc.
  * All rights reserved.
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 const base = require('pwa-kit-dev/configs/jest/jest.config.js')
 
 module.exports = {
-    ...base
+    ...base,
+    coverageThreshold: {
+        global: {
+            branches: 0,
+            functions: 0,
+            lines: 0,
+            statements: 0
+        }
+    },
+    collectCoverageFrom: [
+        'app/**'
+    ]
```

Comment:
Curious, Why are we running a basic 200 test on the `template-express-minimal`, but we are not running the same test on the `template-typescript-minimal`?

Reply:
Per previous discussion, the priority for this ticket is to focus on the tooling, not on the templates.